README.md

```diff
@@ -97,7 +97,7 @@ make setup
 
 Create a super user account for administration purposes.
 ```
-docker-compose run django python app/manage.py createsuperuser
+docker-compose run django python manage.py createsuperuser
```

Comment:
I think we need to update the workdir in the Dockerfile.dev for this to work

Reply:
it already is ;) https://github.com/unicef/iogt/pull/92/files#diff-86930c95a19b82f7e64a962a0053d44e855824813019b3698eae4917a90cdcac

I added some general improvements to docker configuration